.github/workflows/build-and-test.yaml

```diff
@@ -1,72 +1,27 @@
-on:
-  push:
-  pull_request:
-    branches:
-      - master
-  workflow_dispatch:
+name: Build
 
-jobs:
-  build:
-
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.9
-
-      - name: Install pip and development dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements_develop.txt
-
-      - name: Build package
-        run: |
-          python -m cython quadprog/quadprog.pyx
-          python -m build --sdist
-
-      - name: Upload package
-        uses: actions/upload-artifact@v2
-        with:
-          name: sdist
-          path: dist/quadprog*.tar.gz
-
-  test:
```

Comment:
Same comment for ``test``.

Eliminating redundancies can also be done in a subsequent PR :)